setup.py

```diff
@@ -137,6 +137,7 @@ def link_dylib_lib(self, objects, output_libname, output_dir=None,
     PKG_DATA.append(os.path.join('src', 'orig', 'solpos', '*.*'))
     PKG_DATA.append(os.path.join('src', 'orig', 'spectrl2', '*.*'))
 elif not LIB_FILES_EXIST:
+    from solar_utils.tests import test_cdlls
```

Comment:
Better to wrap this in a `try:except` block. This patch solves the issue of installs using wheels when the user doesn't have Numpy, but I believe this will still fail if Numpy or Nose is not installed for installs that build from source.

Another option to consider is adding `install_requires=['nose>=1.3.7', 'numpy>=1.8.1']and possibly `tests_requires` to the setup file so that in installs Numpy and Nose by default. I think that would work, perhaps in addition to the `try:excpt` block. But it depends if you want to make those requirements, maybe you don't want SolarUtils to depend on Numpy or Nose, since they're not used anywhere else in the package.

Reply:
Thanks @mikofski ! Quick questions:
- In which cases would that import fail?
- since "SolarUtils has no requirements for usage" I would be in favor of adding it to ``test_requires`` if we had to choose
- how difficult or easy would it be to remove the dependency on numpy and nose altogether?

Reply:
Also, is there any reason why we didn't enable universal builds for this package?

Reply:
>Also, is there any reason why we didn't enable universal builds for this package?

No, I don't think so because this package uses extensions that are platform and Python specific.

>In which cases would that import fail?

If the user downloads a source distribution or clones the repo, and tries to install or build, then `setup.py` will [call the tests](../blob/e03368caccd8158d71c745223dfecf71cea0eb5d/setup.py#L172-L173)

>how difficult or easy would it be to remove the dependency on numpy and nose altogether?

just rewrite [the tests](../blob/master/solar_utils/tests/test_cdlls.py) to not use nose or numpy.

>since "SolarUtils has no requirements for usage" I would be in favor of adding it to `test_requires` if we had to choose

yah, I agree, `test_requires` is better, and then I would remove the [test calls in `setup.py`](../blob/e03368caccd8158d71c745223dfecf71cea0eb5d/setup.py#L172-L173) - Actually this seems like the right idea, then you can also just remove the [failing import from `setup.py`](../blob/master/setup.py#L20) and problem solved! Then ignore [my other comment on that line](#discussion_r258781138)